src/Cms/Language.php

```diff
@@ -548,9 +559,9 @@ public function toArray(): array
 	}
 
 	/**
-	 * Returns the translation strings for this language
+	 * Returns the language translations object for this language
 	 */
-	public function translations(): array
+	public function translations(): LanguageTranslations
```

Comment:
`Suggestion:` What if we keep returning an array here? Then we would not need to change the `AppTranslations.php` and externally it would work as before.